src/controllers/competition.controller.ts

```diff
@@ -137,22 +148,37 @@ export class CompetitionController {
       // Get all teams (excluding admin teams)
       const teams = await services.teamManager.getAllTeams(false);
       
-      // Map team IDs to names
-      const teamMap = new Map(teams.map(team => [team.id, team.name]));
+      // Filter out disqualified teams and track if any were removed
+      const activeTeams = teams.filter(team => !team.disqualified);
+      const teamMap = new Map(activeTeams.map(team => [team.id, team]));
       
-      // Format leaderboard with team names
-      const formattedLeaderboard = leaderboard.map((entry, index) => ({
-        rank: index + 1,
-        teamId: entry.teamId,
-        teamName: teamMap.get(entry.teamId) || 'Unknown Team',
-        portfolioValue: entry.value
-      }));
+      // Track disqualified teams and create filtered leaderboard
+      const disqualifiedTeamIds = new Set(
+        teams
+          .filter(team => team.disqualified)
```

Comment:
just a thought, but should we use active: true/false rather than disqualified? that way, the _reason_ for being active or not could be flex in the future?

Reply:
yeah good idea - implemented here: [bde5669](https://github.com/recallnet/trading-simulator/pull/10/commits/bde566950c23c546ec676e2bca5f3a2aaf82d550)